js/ui/toast/ui.toast.js

```diff
@@ -176,12 +176,10 @@ const Toast = Overlay.inherit({
     },
 
     _renderContentImpl: function() {
-        if(this.option('message')) {
-            this._message = $('<div>')
-                .addClass(TOAST_MESSAGE_CLASS)
-                .text(this.option('message'))
-                .appendTo(this.$content());
-        }
+        this._message = $('<div>')
```

Comment:
Why we should render message element here? Let's add method for it. And let's in optionChange method add follow code:

```
if(this._message) {
        this._message.text(args.value);
} else if (args.value) {
        this._yourRenderMessageMethod(args.value);
}
```

Reply:
I think that there should be the single place where we render markup, and option change should only change the text